src/Driver/Driver.jl

```diff
@@ -366,6 +368,8 @@ Recognized keyword arguments are:
         run for the specified time (in simulation seconds)
 - `fixed_number_of_steps::Int = -1`:
         if `≥0` perform specified number of steps
+- `degree::NTuple{2, Int} = -1,-1`:
```

Comment:
```suggestion
- `degree::NTuple{2, Int} = (-1, -1)`:
```

Reply:
Thanks! Also done in [5194752](https://github.com/CliMA/ClimateMachine.jl/pull/1845/commits/519475263f3b5476e35fe225bbcd1c9aa853fbf0)